### ISSUE TITLE
Syntax error near unexpected token `('

### ISSUE THREAD

**Issue by emmahsax** [jackw01/npm-license-scraper#4]
I got a syntax error running this:
```
$ npm-license-scraper         
/Users/emmasax/.nvm/versions/node/v16.2.0/bin/npm-license-scraper: line 1: syntax error near unexpected token `('
'Users/emmasax/.nvm/versions/node/v16.2.0/bin/npm-license-scraper: line 1: `// Copyright 2019 jackw01. Released under the MIT License (see LICENSE for details).
```

I am running npm version 7.13.0, node v16.2.0, macOS, zsh. I followed the installation instructions in the README. Is there a syntax error, or am I running something wrong?

**Comment by jackw01** [comment]
Try editing `/Users/emmasax/.nvm/versions/node/v16.2.0/bin/npm-license-scraper` and adding `#!/usr/bin/env node` as the first line. This should fix your issue.

**Comment by emmahsax** [comment]
That worked. I also ran into this issue:
```
env: node\r: No such file or directory
```
Which is described [here](https://stackoverflow.com/questions/30344858/node-script-executable-not-working-on-mac-env-node-r-no-such-file-or-directo). I ended up using the vim solution:
```
vim script

:se ff=unix
:wq
```
And then it all worked perfectly in one of my projects. Interestingly, the other project that I tried to run it in says that there's no dependencies found.... not exactly sure why that is, but 🤷🏼‍♀️
